docs/conf.py

```diff
@@ -152,6 +152,9 @@ def setup(app):
     'cmake': ('https://cmake.org/cmake/help/latest', None),
 }
 
+manpages_url = 'https://linux.die.net/man/{section}/{page}'
```

Comment:
Is there an official source we can link to?

Reply:
It's a common one that pops up in searches and looks like they've been around awhile, but yes, I agree a distro's site would be better. I originally tried Ubuntu, but weirdly it redirects `sched_get_priority_max` to the POSIX section 3 entry instead of the Linux-specific section 2 entry, which it's still doing:

https://manpages.ubuntu.com/manpages/en/man2/sched_get_priority_min
https://manpages.ubuntu.com/manpages/en/man2/sched_get_priority_max

Looking at Debian now though, it looks like it works fine:

https://manpages.debian.org/sched_get_priority_min(2)
https://manpages.debian.org/sched_get_priority_max(2)